sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -66,6 +66,43 @@ export class ChangeFeedIterator<T> {
     get hasMoreResults(): boolean;
 }
 
+// @public
+export interface ChangeFeedIteratorOptions {
+    // Warning: (ae-forgotten-export) The symbol "ChangeFeedResource" needs to be exported by the entry point index.d.ts
+    changeFeedResource?: ChangeFeedResource;
+    // Warning: (ae-forgotten-export) The symbol "ChangeFeedStartType" needs to be exported by the entry point index.d.ts
+    changeFeedStartType?: ChangeFeedStartType;
+    maxItemCount?: number;
+    sessionToken?: string;
+}
+
+// @public
+export class ChangeFeedIteratorResponse<T> {
+    get activityId(): string;
+    get continuationToken(): string;
+    readonly count: number;
+    headers: CosmosHeaders;
+    get requestCharge(): number;
+    readonly result: T;
+    get sessionToken(): string;
+    readonly statusCode: number;
+    readonly SubStatusCode?: number;
+}
+
+// @public
+export abstract class ChangeFeedIteratorV2<T> {
+    // (undocumented)
+    abstract fetchAllFeedRanges(): Promise<void>;
```

Comment:
What is the reason for these fetchAllFeedRanges / fetchContinuationTokenFeedRanges / featchOverlappingFeedRanges methods? Are they public? If yes, why? How is the user expected to use it?

Reply:
Removed these. Earlier these were getting used interally for filling up the queue outside the constructor.